tools/run/solo5-run-virtio.sh

```diff
@@ -80,7 +80,13 @@ while true; do
         ;;
     -n)
         NETIF="$2"
-        ifconfig ${NETIF} >/dev/null || die "no such network interface: ${NETIF}"
+        # Check dependencies
+        type ip &>/dev/null ||
+            type ifconfig &>/dev/null ||
```

Comment:
`type` is provided in `/usr/bin` on FreeBSD so that's fine. You're right about `&>` being a (presumably) bashism. I'll fix that.